packages/form/src/label-wrap.ts

```diff
@@ -81,7 +81,7 @@ export default defineComponent({
           slots.default?.(),
         )
       } else {
-        return h('div', { ref: el }, slots.default?.())
+        return slots.default?.()
```

Comment:
Why did you modify this line anyway? After this modification, how do you think that we can using ref to access the element?

Reply:
you can see [docs](https://element-plus.org/#/en-US/component/form) Inline form
first, inline mode is no need div to wrap label
then, i think the label width is computed in form-item,inline mode is no need this ref

Reply:
The definition of inline mode is not who the DOM tree looks like, is how we represent it in the UI frame.

Reply:
so you think inline mode also needs div wrap? at least you need to change the style

Reply:
I double checked the V2 version, it indeed does not have any 'div' wrapped the label. As far as I could tell, enabling  `inline` mode, the method `getLabelWidth` still get called because it only depends on `isAutoWidth`

Reply:
> so you think inline mode also needs div wrap? at least you need to change the style

What about I pass down a template with structure like
```vue
<div>
  <label>
  </label>
</div>
```

Reply:
Maybe wrap the `$slots.default` with a `div` with `inlined` display property is more flexible on my understanding.

Reply:
Besides this, I've got no questions about this PR

Reply:
> Maybe wrap the `$slots.default` with a `div` with `inlined` display property is more flexible on my understanding.

I don’t know if my understanding is correct，if do this, the inner label with `class="el-form-item__label" :style="labelStyle"` is lose it's effect